### DISCUSSION TITLE
Can the simulated data be exported ?

### DISCUSSION THREAD

**Discussion by nyue** [Genesis-Embodied-AI/genesis-world#301] [Q&A]
The use case is for creative people to be able to import the simulated data e.g. meshes, flow data, and import them into DCC e.g. Blender, Houdini, Maya, for further creative story telling.

**Comment by zhouxian** [comment]
Yes sounds useful. We can design some scene level description file or support USD

**Reply by nyue** [reply]
Happy to participate in the design so I can test and feedback to the team.

**Comment by TwinSnakes007** [comment]
> Yes sounds useful. We can design some scene level description file or support USD

USD would be awesome!
